app/src/main/kotlin/edu/stanford/bdh/engagehf/bluetooth/data/models/VitalDisplayUiState.kt

```diff
@@ -0,0 +1,24 @@
+package edu.stanford.bdh.engagehf.bluetooth.data.models
+
+import edu.stanford.bdh.engagehf.bluetooth.component.OperationStatus
+
+data class VitalDisplayUiState(
```

Comment:
This object shouldn't be needed, I would use the three properties (VitalDisplayData) directly in the main ui state of the view model instead for each type